packages/shared/src/components/onboarding/Plus/OnboardingPlusVariationV1.tsx

```diff
@@ -0,0 +1,251 @@
+          {price}
+        </Typography>
+        <Typography
+          color={TypographyColor.Tertiary}
+          type={TypographyType.Footnote}
+        >
+          /month
+        </Typography>
+      </div>
+    </div>
+  );
+};
+
+export const OnboardingPlusVariationV1 = ({
+  onClickNext,
+  onClickPlus,
+  headline = 'Suffer less. Debugging bad decisions is harder.',
+}: OnboardingStepProps): ReactElement => {
+  const isLaptop = useViewSize(ViewSize.Laptop);
+  const { productOptions } = usePaymentContext();
+
+  const { logEvent } = useLogContext();
+  const item = useMemo(
+    () =>
+      productOptions.find(
+        ({ metadata }) => metadata.appsId === PlusPriceTypeAppsId.Annual,
+      ),
+    [productOptions],
+  );
+
+  const featureCardsNew = plusFeatureList.filter(
+    (plusItem) => plusItem.status === PlusItemStatus.Ready,
+  );
+
+  const handleItemHover = (hoverItem: PlusItem) => {
+    logEvent({
+      event_name: LogEvent.HoverPlusFeature,
+      target_id: hoverItem.id,
+      target_type: TargetType.Grid,
+    });
+  };
+
+  const items = featureCardsNew.map((plusItem) => {
+    return (
+      <div
+        key={plusItem.id}
+        className="w-full"
+        onMouseEnter={() => handleItemHover(plusItem)}
+      >
+        <div className="mb-6 h-50 w-full overflow-hidden rounded-10">
+          {plusItem.modalProps?.mediaType === 'video' ? (
+            <video
+              className="w-full border-none"
+              poster={plusItem.modalProps.imageUrl}
+              src={plusItem.modalProps.videoUrl}
+              muted
+              autoPlay
+              loop
+              playsInline
+              disablePictureInPicture
+              controls={false}
+              title={plusItem.modalProps.title}
+            />
+          ) : (
+            <img
+              src={plusItem.modalProps?.imageUrl}
+              alt={plusItem.modalProps?.title}
+              className="h-50 w-full rounded-10"
+            />
+          )}
+        </div>
+        <Typography type={TypographyType.Title3} bold className="mb-3">
+          {plusItem.label}
+        </Typography>
+        <Typography
+          type={TypographyType.Callout}
+          color={TypographyColor.Tertiary}
+        >
+          {plusItem.modalProps?.description}
+        </Typography>
+      </div>
+    );
+  });
+
+  return (
+    <section className="flex w-full max-w-screen-laptop flex-col gap-6 tablet:gap-10 tablet:px-10">
+      <header className="text-center">
+        <Typography
+          bold
+          tag={TypographyTag.H1}
+          type={isLaptop ? TypographyType.LargeTitle : TypographyType.Title2}
+          className="mb-4 tablet:mb-6"
+        >
+          {headline}
+        </Typography>
+      </header>
+
+      <div className="flex w-full flex-col gap-10 tablet:flex-row tablet:gap-4">
+        <div className="mx-auto flex flex-1 flex-col gap-4">
+          {/* Plus Plan Card */}
+          <a
+            href="#"
```

Comment:
empty href correct?

Reply:
Yeah it's just a hijack click basically
We use it too often already in onboarding :(